python/dgl/graphbolt/minibatch_sampler.py

```diff
@@ -2,16 +2,67 @@
 
 from collections.abc import Mapping
 from functools import partial
-from typing import Iterator, Optional
+from typing import Callable, Iterator, Optional
 
 from torch.utils.data import default_collate
 from torchdata.datapipes.iter import IterableWrapper, IterDataPipe
 
 from ..batch import batch as dgl_batch
 from ..heterograph import DGLGraph
+from .data_block import LinkPredictionBlock, NodeClassificationBlock
 from .itemset import ItemSet, ItemSetDict
 
-__all__ = ["MinibatchSampler"]
+__all__ = [
+    "MinibatchSampler",
+    "mapper_node_classification",
+    "mapper_node_classification_hetero",
+]
+
+
+def _mapper_default(batch):
+    """Default mapper.
+    [TODO] This is a temporary solution to be compatible with current cases.
+    It will be deprecated in the near future.
+    """
+    return batch
+
+
+def mapper_node_classification(batch):
```

Comment:
What about moving all this to data_block.py, all this are just initiation function of datablock?